music-journal/components/Login.js

```diff
@@ -1,29 +1,36 @@
-import { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import Head from 'next/head';
-import { auth, firebase } from '../utils/initFirebase';
+import { getMe, deleteMe } from '../utils/auth';
 import router from 'next/router';
 import { Container, Row, Button, Image } from 'react-bootstrap/';
 
-const Login = ({ user }) => {
+const Login = () => {
+  const [user, setUser] = useState()
 
   const handleSignIn = () => {
     router.push(`/api/spotify-auth`);
   }
 
   //This should probably move into the nav
-  const handleSignOut = () => {
-    auth.signOut().then(function () {
-      alert('Logout successful');
-    }).catch(function (error) {
-      alert('OOps something went wrong check your console');
-      // console.log(err);
-    });
+  const handleSignOut = async () => {
+    const success = await deleteMe()
+    if (success) {
+      console.log('success', success)
+      setUser()
+    } 
   }
 
   useEffect(() => {
-    if (user) {
-      router.push('/');
+    const getUser = async () => { 
+      try {
+        const [, currentUser] = await getMe()
```

Comment:
so this essentially throws away the token variable?

Reply:
yeah I don't think we need it so you just ignore the first item in the array